client/a8c-for-agencies/sections/partner-directory/types.ts

```diff
@@ -1,13 +1,17 @@
+export type AgencyDirectoryApplicationStatus = 'pending' | 'in-progress' | 'completed';
+
+export type DirectoryApplicationType = 'wordpress' | 'jetpack' | 'woocommerce' | 'pressable';
+
 export interface AgencyDirectoryApplication {
-	status: 'pending' | 'in-progress' | 'completed';
+	status?: AgencyDirectoryApplicationStatus;
```

Comment:
If I recall correctly, in the end, we leave this as a required parameter.

Reply:
Good catch. I've made the necessary adjustments to resolve this. 👍